lambda/voting/README.md

```diff
@@ -70,6 +70,16 @@ npm run start
 
 The demo application allows you to register and vote for the candidate.
 
+# Get Candidate Details
```

Comment:
we can actually remove this. We don;t have any other UI details in the README

Reply:
done